src/main/java/org/openstreetmap/atlas/geography/atlas/routing/AllPathsRouter.java

```diff
@@ -193,7 +193,7 @@ private static void allRoutes(final Edge start, final Edge end, final Stack<Edge
         }
         // Add start edge to the path
         path.push(start);
-        onPath.add(start.getIdentifier());
```

Comment:
Can you add a comment on this method saying that this won't be giving routes which will have a set of same positive and negative edge